src/palace/manager/service/google_drive/google_drive.py

```diff
@@ -86,13 +86,14 @@ def create_file(
             parents = [parent_folder_id] if parent_folder_id else []
             file_metadata = dict(name=file_name, parents=parents)
             file = (
-                self.service.files()
-                .create(body=file_metadata, media_body=media, fields="*")  # type: ignore[arg-type]
+                self.service.files()  # type: ignore[attr-defined]
+                .create(body=file_metadata, media_body=media, fields="*")
                 .execute()
             )
         finally:
             stream.close()
 
+        assert file
```

Comment:
Minor: Did you mean to add this assertion? Might want to attach a message to it, or at least add a comment mentioning why the assertion is here.

Reply:
Yes: intentional and I add a message.